GrandCentralBoard/Widgets/Bonus/Scene/Bubble.swift

```diff
@@ -9,15 +9,17 @@ class Bubble: SKSpriteNode, BubbleScalingAnimatorDelegate {
     
     private let initialSize = CGSize(width: 140, height: 140)
     private var lastBonusDate: NSDate? = nil
+    private let bubbleResizeDuration: NSTimeInterval
     private lazy var scalingAnimator: BubbleScalingAnimator = {
-        let animator = BubbleScalingAnimator(spriteNode: self)
+        let animator = BubbleScalingAnimator(spriteNode: self, bubbleResizeDuration: self.bubbleResizeDuration)
         animator.delegate = self
         return animator
     }()
     
-    init(bubbleViewModel: BubbleViewModel) {
+    init(bubbleViewModel: BubbleViewModel, bubbleResizeDuration: NSTimeInterval) {
```

Comment:
`bubbleResizeDuration` should be (IMO) a part of the `bubbleViewModel` - view models configures the view and `bubbleResizeDuration` is a configuration parameter.